src/topics/command-line-tool.md

```diff
@@ -10,6 +10,9 @@ The following example contains a minimal example of a CWL
 command-line tool for the `echo` Linux command, using inputs and
 outputs.
 
+How Command-lines work
+A Command Line Tool is a non-interactive executable program that reads some input, performs a computation, and terminates after producing some output. Command line programs are a flexible unit of code sharing and reuse, unfortunately the syntax and input/output semantics among command line programs is extremely heterogeneous. A common layer for describing the syntax and semantics of programs can reduce this incidental complexity by providing a consistent way to connect programs together. This specification defines the Common Workflow Language (CWL) Command Line Tool Description, a vendor-neutral standard for describing the syntax and input/output semantics of command line programs.
```

Comment:
This looks like it is quoting something, if so, we need it to be properly attributed. I would prefer a shorter paraphrase if possible

Reply:
Hello @swzCuroverse thank you for the feedback, i have made a commit with changes.